src/generators/mappersGenerator.ts

```diff
@@ -171,7 +195,14 @@ function writeModelProperties(
     //  color: { type: { name: "String" }, serializedName: "color" }
     // }
     writeParentMappers(parents, writer);
-    writeObjectProps(modelProperties, writer);
+    // Write all sub-mappers
+    if (modelProperties) {
```

Comment:
Added this to make sure we correctly generate mappers for each modelProperty, otherwise we just stringify and miss any custom handling